tvos.yml

```diff
@@ -16,7 +16,6 @@
         :source: $HE_LIBOQS_SOURCE
         :tag: $HE_LIBOQS_TAG
       :build:
-        - git apply ../../liboqs/0001-Prefer-arc4random-on-Apple-platforms-1544.patch || true
```

Comment:
Let's document here that this removal was supposed to be done in https://github.com/expressvpn/lightway-core/pull/129 but we forgot back then.

Makes sense to remove this now!